protocol-designer/src/components/StepEditForm/fields/ProfileItemRows.tsx

```diff
@@ -309,11 +308,17 @@ const ProfileStepRow = (props: ProfileStepRowProps) => {
     )
   }
 
-  const deleteProfileStep = () => {
+  const deleteProfileStep = (): void => {
     dispatch(steplistActions.deleteProfileStep({ id: profileStepItem.id }))
   }
-  const names = ['title', 'temperature', 'durationMinutes', 'durationSeconds']
-  const units = {
+  type Names = 'title' | 'temperature' | 'durationMinutes' | 'durationSeconds'
+  const names: Names[] = [
+    'title',
+    'temperature',
+    'durationMinutes',
+    'durationSeconds',
+  ]
```

Comment:
```suggestion
  ] as const
```
this should remove the need for the `Names` type